packages/react-admin-core/src/table/paging/createPagePagingActions.ts

```diff
@@ -0,0 +1,26 @@
+import { IPagingApi } from "../useTableQueryPaging";
+import { IPagingInfo } from "./IPagingInfo";
+
+interface IPagePagingData {
+    nextPage: number | null;
+    previousPage: number | null;
+    totalPages?: number;
+}
+
+export function createPagePagingActions<TData extends IPagePagingData>(pagingApi: IPagingApi<number>, data: TData): IPagingInfo {
```

Comment:
What's the purpose of this function? Should this be used for client-side paging? There is no corresponding example for it.

Reply:
For APIs that have paging using a number page parameter. Very similar to createRestPagingActions, that additionally parses page numbers from links.